### ISSUE TITLE
ADA Violation - required ARIA attribute not present

### ISSUE THREAD

**Issue by david-kirkland** [angular-ui/ui-select#1775]
### Bug description:
ARIA widget roles must have appropriate attributes describing the state or properties of the widget. The `<input role="combobox" ...>` element should include the `aria-expanded` attribute.  The absence of this ARIA attribute results in a WCAG2 (A) violation.

### Link to minimally-working plunker that reproduces the issue:
This FireFox plugin will allow you to test accessibility...
https://addons.mozilla.org/en-us/firefox/addon/axe-devtools/

### Version of Angular, UI-Select, and Bootstrap:
Angular: 1.5.2
Bootstrap: 1.2.5
UI-Select:  0.18.1

**Comment by user378230** [comment]
You could submit a PR for this. 

Seems simple enough and I doubt anyone else would ever get to it...

**Comment by david-kirkland** [comment]
Done... [https://github.com/angular-ui/ui-select/pull/1776](url)
